kinder/ci/tools/update-workflows/templates/workflows/rootless-tasks.yaml

```diff
@@ -0,0 +1,277 @@
+# IMPORTANT! this workflow is imported by regular-* workflows.
+version: 1
+summary: |
+  This workflow implements a sequence of tasks used test the proper functioning
+  of kubeadm version X with Kubernetes version X.
```

Comment:
we should add a description related to rootless here

Reply:
Done. Thanks!